<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QProgressBar" name="UpdateProgressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>631</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="TargetFolderLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>521</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectTargetFolderButton">
   <property name="geometry">
    <rect>
     <x>538</x>
     <y>400</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="installPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>631</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Install</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>631</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>135</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="AcrobatReader_checkBox">
      <property name="text">
       <string>Acrobat Reader</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="AnyDesk_checkBox">
      <property name="text">
       <string>AnyDesk</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Teams_checkBox">
      <property name="text">
       <string>Teams</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="Java_checkBox">
      <property name="text">
       <string>Java</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ShipSure_checkBox">
      <property name="text">
       <string>ShipSure</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="zip_checkBox">
      <property name="text">
       <string>7Zip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="SelectAll_checkBox">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
